### ISSUE TITLE
I lost 3000 Iota using the Iota light wallet 2.5.7

### ISSUE THREAD

**Issue by adenluis** [iotaledger/wallet#1158]
On the 18/02/2018 I tried to send my Iota to the Binance exchange from my light wallet that I had recently upgraded from Github to the 2.5.7 version. 

The Binance Iota address was: 	BYTFJXWIEM9QYPOYWLDLDJJPTREQGOKSKGOETPMQVWCUEWXJNAAQBGAOT9LIQCLOTDXHZF9QFK9UATSXDBYVSYLDS9

I tried to “promote/re-attach” but still nothing was happening, I waited 3 days with 0/1confirmations on Binanace. Than I went back to my light wallet and notice I still had my 3.0+Gi there. I assumed the transaction must have bounced back to my light wallet. 

On the 21/02/2018 I sent my Iota “again” from my light wallet to my second Binance account to see if that would work. 

This is my second Binance Iota address: CG9PCOJNRIDODHJ9YBBONBPUMMGOUKSAGAPMYRAZ9DBHGFMQDBLHGKTYK9HWGGWKQTDPG9NMCUFKYMGMDQGVYSJZGW

This time I thought I would try my best to push this transaction through, I tried to reach out to people in the community running full nodes to see if they could help, I tried to “promote/re-attach” straight away. 

Than I came across this site: http://www.reattach.online/

It asked me to paste either my Hash/Address in to “promote/re-attach”.

Everyone 1 minute it would try to re-attach to the tangle.

Before I knew it I had over 69 transactions recorded in my Iota wallet, but the out going address was different. 

Instead of my Iota going to my second Binance address, it went to this address.

T9AWPYLQRY9QPNCKVNHOEOOYZVMHOWMWWFDZBGPGNTJSVUFPRSSRNKKELXULFRHTRMDUMUYDJVCPMPOLZUFJEVLFSW

In the end I reached out to the developer of this site http://www.reattach.online/ where he concluded this.

Your case is the following:
1. You sent iotas from a synced node.
2. You entered in an unsynced node and sent iota again. Unsynced node didn't know about your past transaction, then the wallet didn't advice you about the problem, and let you make another transaction.

I still don’t understand how a simple mistake like this is not protected by the wallet/network if this is the case. Can anyone help me understand this better? And have I definitely lost my Iota trying to make a transaction from the light wallet to Bianance?

Any help would be super appreciated !

**Comment by CaliBitcoins** [comment]
Hi there.... I see a lot of people are missing IOTA. I am one of them. Has IOTA done anything to help you resolve or reclaim lost funds from incomplete transactions/hacks?

**Comment by adenluis** [comment]
Hey, I have not managed to find anyone who can help, I have tried multiple platforms to try reach out and nobody can even explain how I lost the 3,000 iota doing a simple transaction.

So I have just accepted the fact that I am not getting it back. Really sucks :(

If you find anyone that could help please let me know

**Comment by wonkytonky** [comment]
from what i can see : you some how tried to send money from the same address 2 times to another address.     this makes the address security weak to attacks ( dont spend more then 1 time from 1 address).
so chances are  a hacker screening for this  kind of addresses will take the money. 
tldr : money is gone 

but i wonder how you could double spend did you use android wallet  too? normaly github wallet should warn you about this .

maybe because of this special case the wallet didnt warn you 
![image](https://user-images.githubusercontent.com/19390834/37252350-feab76e2-251f-11e8-9d01-1232e92c93c9.png)

devs might need to look at this .

**Comment by adenluis** [comment]
Thanks for looking into it, really appreciate it. How would I contact the dev team?

I'm not sure what you mean by Android wallet, I had the latest wallet version downloaded to my Mac if thats what you mean?

**Comment by wonkytonky** [comment]
there's an android wallet too .. sometimes ppl use it to doublespend since github wallet does not allow it..  althoug i dont know if android wallet is safe to use these days.. but apparently you didnt use it..

i'm sure devs read the issues here .. if they find someting that's a real bug they will fix it.

**Comment by adenluis** [comment]
Yeah I have never seen that message before.

I have been trying to get my iota out of my light wallet for over 2 months before I lost it. The first time I sent my iota to Binance the transaction sat there unconfirmed for over 3 weeks, then I think Binance rejected the transaction because it did not say '0/1 confirmations' under 'deposits' anymore it was just gone, and the funds where back in my wallet. Then I tried again about a month later and that's what happen with the above story I wrote on this page.

Can't say I would be recommending iota to anyone if the safest place for it is the exchange.

**Comment by adenluis** [comment]
Thanks for helping out anyway wonkytonky, still a little confused with how I lost it all using the iota wallet but I really appreciate you trying to help!

**Comment by jpenner123** [comment]
My goodness...I have been using IOTA wallet for quite some time and never had any issues. Yes, sometimes transactions get stuck... but the loss of 3000 IOTA is really bad... sorry to hear that dude...

**Comment by adenluis** [comment]
Yeah I was pretty devastated, I feel like iota has a long way to go before it will ever be used in real world cases, over 3 months I had my iota stuck in that wallet, and when it finally came out it went into someone else's wallet. 

Not an ideal form of currency

**Comment by CaliBitcoins** [comment]
Sorry to hear that @adenluis and thank you for your assistance @wonkytonky ... I am setting up a website to track these fraudulent transactions after losing my funds too. Hopefully in time we can recover these damages. Good luck!

**Comment by adenluis** [comment]
That's a great idea @CaliBitcoins. If there is anyway I can still recover the iota I lost I would be most appreciative! Also if there is anyway I can help please don't hesitate to ask.

**Comment by wonkytonky** [comment]
iota team is working with police to find stolen funds...   by those online seed generator hackers..  I know they are working on that.

**Comment by CaliBitcoins** [comment]
I spoke to David from IOTA, he denies any security defects in this system (of course), but I found this article from MIT's Crypto Initiative Project:

https://medium.com/@neha/cryptographic-vulnerabilities-in-iota-9a6a9ddc4367

**Comment by changsuh** [comment]
I got iota coin 332.167 from binance, I sent to my iota wallet on Dec.21.2017.
I was checking  iota 332.167 in the wallet,Just I was keeping over 3 month,
When I was checking around 10 days ago, I couldn't see any coin in iota my wallet,
Please adivice !!

**Comment by novagalaktika** [comment]
Кошелек Iota Light 2.5.7 Баланс равен 0 -----

**Comment by changsuh** [comment]
Hi, nux 1103
I got same problem,Can you give me information,
How did get iota again,
